modules/ROOT/pages/tutorial-setup.adoc

```diff
@@ -4,7 +4,36 @@ Those tutorials are written targeting a Linux environment with a working `libvir
 
 == Downloading a Fedora CoreOS QCOW2 image
 
-First of all, we have to download the latest stable release of Fedora CoreOS. As we will use `libvirt`, we will download an archive containing a QEMU image from link:https://getfedora.org/en/coreos/download?tab=metal_virtualized&stream=stable[getfedora.org]:
+First of all, we have to download the latest stable release of Fedora CoreOS. As we will use `libvirt`, we will download a QEMU image.
+
+=== Downloading with `coreos-installer` on Fedora
+
+If you are using Fedora, you can install `coreos-installer` and directly download the latest stable image with:
+
+[source,bash]
+----
+sudo dnf install -y coreos-installer
```

Comment:
Do we want to keep this? Isn't the podman flow an overall better superset of this?

Reply:
I agree, but at the same time we have it as we need it for the live ISOs, so why not? Maybe I should put it lower?

Reply:
With:
```
podman run --rm -ti -v .:/data:Z --workdir /data quay.io/coreos/fcct --pretty --strict fcct-updates.yaml --output updates.ign
```
we also have a compelling option for running fcct from a container "just-like" a regular one with current directory access.